tests/test_client.py

```diff
@@ -179,6 +180,82 @@ def create(self, stream, ext_obj=None, start_date=None):
     def make_id(self, stream):
         return '#{}_{}'.format(stream, datetime.now().strftime('%Y%m%d%H%M%S%fZ'))
 
+    def get_catalog_object(self, obj_id):
+        response = self._client.catalog.retrieve_catalog_object(object_id=obj_id)
+        if response.is_error():
+            print(response.body.get('errrors'))
+        return response.body.get('object')
+
+    def get_an_inventory_adjustment(self, obj_id):
+        response = self._client.inventory.retrieve_inventory_changes(catalog_object_id=obj_id)
+
+        if response.is_error():
+            raise RuntimeError('GET INVENTORY_ADJUSTMENT: {}'.format(response.errors))
+        return response
+
+    def create_inventory_adjustment(self, start_date=None):
+        # Create an item
+        item = self.create_item().body.get('objects')[0]
+
+        # Crate an item_variation and get it's ID
+        item_variation = self.create_item_variation(item.get('id')).body.get('catalog_object')
+        catalog_obj_id = item_variation.get('id')
+
+        # Get a random location
+        all_locations = self.get_all('locations')
+        loc_id =  all_locations[random.randint(0, len(all_locations) - 1)].get('id')
+        from_state = 'IN_STOCK'  # inventory_obj.get('state')
+
+        # Adjustment logic
+        made_id = self.make_id('inventory')
+        if from_state == 'IN_STOCK':
```

Comment:
Isn't this always true?

Reply:
For the moment yes. Ideally we will change it to do different types of transitions.